chat-commands.js

```diff
@@ -3329,9 +3326,9 @@ exports.commands = {
 		}
 		if (!target) return this.parse('/help bash');
 
-		connection.sendTo(room, "$ " + target);
+		connection.sendTo(room, `$${target}`);
```

Comment:
Is this actually how that works?

Reply:
I mean, this is certainly correct code for `"$" + target`. It's unfortunately wrong for `"$ " + target`, though.